Legobot/Legos/Help.py

```diff
@@ -49,7 +49,13 @@ def handle(self, message):
             for lego in legos:
                 lego_proxy = lego.proxy()
                 if lego_proxy.get_name().get() == function:
-                    help_str = lego_proxy.get_help().get()
+                    help_obj = lego_proxy.get_help().get()
+                    if isinstance(help_obj, tuple):
+                        if help_obj[1] == 'dm':
```

Comment:
I'm still kicking through this, but my opinion at the moment is that the structure of the tuple should be `(str, bool)` and after thinking through this some more, I'm also thinking that maybe there should be a help object in the same pattern as the message object that defaults certain fields. I definitely don't want to rush this one, though.